packages/iris-grid/src/index.ts

```diff
@@ -8,6 +8,7 @@ export * from './CommonTypes';
 export { default as ColumnHeaderGroup } from './ColumnHeaderGroup';
 export * from './PartitionedGridModel';
 export * from './IrisGrid';
+export { default as IrisGridType } from './IrisGrid';
```

Comment:
Isn't this the lazy component? Why the `Type` suffix?

Reply:
It's just for one spot where we assign a ref and call into `IrisGrid` methods. It is a component (I might be able to export the typeof instead), but it's intended to be used as just a type in that one spot

Reply:
Gotcha. Yeah if you can export as a type, this makes sense. If you can't, I would suggest renaming it to `IrisGridBase` or something of your liking that doesn't imply it is just a type.

Reply:
I exported as type and verified that `new IrisGridBase()` results in a TS error because it was exported w/ the type modifier

It's not intended to be used as its own component (the `IrisGrid` lazy wrapper should just work) and is just because in the panel we call some class methods directly using the ref